### PR TITLE
Components in Drupal fixes

### PR DESCRIPTION
**Pull request by rburkey87**
overrides styling from Zurb Foundations. Fixes search bar and header link styling.

### CODE REVIEW

File: scss/byu_default.scss [hunk 1]
```diff
@@ -26,106 +26,98 @@
...
-// @import "components/foundation-icons";
+/* Components
+ @import "components/buttons";
+ @import "components/type";
+ @import "components/forms";
+ @import "components/foundation-icons";
 
-// Themes
-// @import "theme/button-light";
+ Themes
```
Comment (katrialesser) [inline scss/byu_default.scss]: missing the comment before Themes

File: scss/byu_default.scss [hunk 2]
```diff
@@ -136,7 +128,7 @@ byu-header a:hover,
 }
 
 /* -------- search --------- */
-/* -- from instructions on byu-search component for SIMPLE search -- */
```
Comment (katrialesser) [inline scss/byu_default.scss]: We should keep a comment here explaining that the search styling comes from the byu-search instructions; i know they don't match but once we finalize what they should be we can get it updated in the instructions.

File: scss/byu_default.scss [hunk 3]
```diff
@@ -163,12 +151,11 @@ byu-search.mobile-view .form-item .custom-search-box {
 }
 
 
-
 /* ------------ */
 
-// headings
-h1,h2,/*h3, */h4,h5,h6 {
-  font-family: "Vitesse A", "Vitesse B", Georgia, serif;y;
+/* headings - None of this seems to be applying & the variables need defined
```
Comment (katrialesser) [inline scss/byu_default.scss]: it should be applying now that it is compiling - check and remove comment?

File: template.php [hunk 4]
```diff
@@ -12,8 +12,17 @@ function byu2017_d7_preprocess_html(&$variables) {
 //    drupal_add_css('//cloud.typography.com/75214/6517752/css/fonts.css', array('type' => 'external'));
 
     // To include the components and their styling:
-  drupal_add_css('//cdn.byu.edu/byu-theme-components/latest/byu-theme-components.min.css', array('type' => 'external'));
-  drupal_add_js('//cdn.byu.edu/byu-theme-components/latest/byu-theme-components.min.js', 'external');
+  drupal_add_css('//cdn.byu.edu/byu-theme-components/unstable/byu-theme-components.min.css', array('type' => 'external'));
```
Comment (katrialesser) [inline template.php]: Change back to latest

File: scss/byu_default.scss (hunk 1 above)
Comment (rburkey87) [inline scss/byu_default.scss]: It's within a larger comment block 
`/* Components
 @import "components/buttons";
 @import "components/type";
 @import "components/forms";
 @import "components/foundation-icons";

 Themes
 @import "theme/button-light";

 IE8 Grid Support
@import "ie"; */`

File: scss/byu_default.scss (hunk 3 above)
Comment (rburkey87) [inline scss/byu_default.scss]: removed comment

File: scss/byu_default.scss (hunk 2 above)
Comment (rburkey87) [inline scss/byu_default.scss]: done

File: template.php (hunk 4 above)
Comment (rburkey87) [inline template.php]: done

**Pull request merged**